optimum/exporters/onnx/model_configs.py

```diff
@@ -192,6 +192,11 @@ class GPTNeoOnnxConfig(TextDecoderOnnxConfig):
     NORMALIZED_CONFIG_CLASS = NormalizedTextConfig.with_args(num_attention_heads="num_heads")
 
 
+class GPTNeoXOnnxConfig(TextDecoderOnnxConfig):
+    DEFAULT_ONNX_OPSET = 13
+    NORMALIZED_CONFIG_CLASS = NormalizedTextConfig.with_args(num_attention_heads="num_attention_heads")
```

Comment:
```suggestion
    NORMALIZED_CONFIG_CLASS = NormalizedTextConfig
```

It is enough as `num_attention_heads` is the default: https://github.com/huggingface/optimum/blob/40a01b3c883ca3c092a4493d3f5ca524ed3109ab/optimum/utils/normalized_config.py#L72

Reply:
Makes sense, updated!